src/containers/Home/HomeHero/index.tsx

```diff
@@ -62,9 +64,14 @@ const HomeHero: FC = () => {
             <div className="HomeHero__social-media-links">{renderSocialMediaLinks()}</div>
           </div>
         </div>
-        <div className="HomeHero__right">
-          <img alt="hero" className="HomeHero__image" src={HeroV2Svg} />
-        </div>
+        {width > 414 && (
```

Comment:
@jamessspanggg Do we need this condition?

Reply:
Previously, it sets the `display: none` when the screen width is less than 414 px. The reason why I added this condition is so that we will not issue the network request to obtain the images if so